docs/Developers/Newcomers_Guide.rst

```diff
@@ -208,27 +208,27 @@ Now you can start working on it!
     You can do amazing stuff using corobo.
 
         * Issue assigning as stated earlier.
-        * File issues::
+        * File issues
 
-            corobo file issue <repo> <title>
-            <description>
+            **corobo file issue <repo> <title>**
+            **<description>**
 
-        * You shouldn't close any PR, instead mark them as work in progress::
+        * You shouldn't close any PR, instead mark them as work in progress
 
-            corobo mark wip <full url>
+            **corobo mark wip <full url>**
```

Comment:
I ud suggest boldning in a different commit <if required>, what you say @Makman2 ?

Reply:
I think that is unnecessary. This could be considered as part of emphasizing and relevant to the commit.

Reply:
@adhikasp (Y)

Reply:
Hm this should be rendered as code, so it should be kept. Imo a different issue.

Reply:
@Makman2 These colons are not for text but to use a different block defined in css. If i keep these colons, the background will be as defined here https://github.com/coala/coala/blob/1db00d706353bb0d8a175c6d561df21e51cad6e7/docs/_static/custom.css#L108 or else the left padding is disturbed.
And that is why, the rendered code also needs to be bold. Additionally, bold text appears better over yellow background. But this i can keep normal for this PR and open a separate issue to bold this text. What do you suggest?